test/unit/lib/cli/interactive-setup/dashboard-set-org.test.js

```diff
@@ -36,11 +36,14 @@ describe('test/unit/lib/cli/interactive-setup/dashboard-set-org.test.js', functi
             return {
               awsAccountId: '377024778620',
               supportedRuntimes: [
-                'nodejs10.x',
                 'nodejs12.x',
-                'python2.7',
-                'python3.6',
+                'nodejs14.x',
+                'nodejs16.x',
+                'nodejs18.x',
                 'python3.7',
+                'python3.8',
+                'python3.9',
+                'python3.10',
```

Comment:
We can't add Python 3.10 here until after https://github.com/serverless/serverless/pull/11922 is merged.